code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -1009,6 +1009,11 @@
 	reload_time = 60
 	reloading_str = "recharging"
 
+/datum/limb/gun/flock_stunner/point_blank(mob/living/target, mob/living/user)
+	if (isflockmob(target))
+		return
+	return ..()
```

Comment:
```suggestion
	return ..()
```
Parent doesn't actually return anything, but not sure if matters

Reply:
well the parent actually returns either null or FALSE seemingly arbitrarily, this seems like the best option to prevent bugs in future in case we ever want the return value to mean something